app/pods/publications/publication/translations/documents/controller.js

```diff
@@ -28,6 +64,12 @@ export default class PublicationsPublicationTranslationsDocumentController exten
     return this.translationSubcase.isFinished;
   }
 
+  @action
+  changeSorting(sortingString) {
+    this.sortingString = sortingString;
+    this.set('qpSortingString', sortingString);
```

Comment:
Waarom heb je beiden nog nodig? Is enkel `qpSortingString` niet voldoende?

Reply:
qp's worden liever niet getracked omdat die performantieissues zou geven.

Reply:
Dat klopt, maar dan heb je voldoende aan de property die geupdate wordt met `set()`. Om te zorgen dat de `get pieces()` geupdate worden wanneer de sortering wijzigt, moet je deze annoteren met `@computed('sort')`. Dit is nog afkomstig uit het pre-Ember Octane tijdperk. Eens we geupgrade zijn en er een `@tracked sort` is, kan de `@computed` annotatie weg.

Ik heb de change gecommit met de nodige TODOs voor later zodanig dat deze PR kan doorstromen naar development.